keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -64,7 +64,7 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
     }
 
     fun delete() {
-      repository.deleteByApplication(deliveryConfig.application)
+      repository.deleteWithoutSerialization(deliveryConfig.application)
```

Comment:
Shouldn't we just remove this side effect without necessarily calling it out in the name of the function?

Reply:
Sure, I'll rename. It was leftover from when I was going to leave both methods. Good call

Reply:
i have such a hard time remembering which is which with serialization/deserialization lol